PrivateHospitals/backend/PrivateHospitals/PrivateHospital.Migration/Services/PatientService.cs

```diff
@@ -11,7 +12,7 @@
 
 namespace PrivateHospital.Migration.Services
 {
-    public class PatientService(IRepository<Patient> _patientRepository)
+    public class PatientService(PatientRepository _patientRepository)
```

Comment:
Change all similar injections. U should understand that you cannot inject a class into another class, only abstract class or interface because u violating DI principle and this code becomes impossible to test